README.md

```diff
@@ -29,6 +29,7 @@ missing please open an issue. If you are interested in joining check out
 - Gibson Fahnestock [@gibfahn](https://github.com/gibfahn)
 - Johan Bergström [@jbergstroem](https://github.com/jbergstroem)
 - João Reis [@joaocgreis](https://github.com/joaocgreis)
+- Jon Moss [@maclover7](https://github.com/maclover7)
```

Comment:
Could you move yourself to between `kunalspathak` and `mhdawson`? I know it's not particularly obvious (https://github.com/nodejs/build/pull/930), but this is sorted by GitHub ID.